dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=qBittorrent-nox service for user %I
 Documentation=man:qbittorrent-nox(1)
-Wants=network-online.target
-After=network-online.target nss-lookup.target
+Wants=network-online.target local-fs.target
```

Comment:
I think adding it here would be a little too strong, other users might not need local FS as they rely on remote mounts.

Reply:
Local filesystems being mounted is a reasonable expectation IMHO. E.g. someone might have /home on a separate partition, and qbittorrent would have problems reading stuff from ~/.config if mounting local filesystem is a bit slow.

I am not sure what the convention when writing systemd units is and if it's normal to wait for local-fs.target, but if the application is using the local filesystem (and qbittorrent sure is using user's $HOME), I'd say that delaying start until local filesystems are mounted is reasonable. Since these are local mounts, the delay shouldn't be really that large like it could be for network mounts (for me, it happens because of HDD spinup, which is just a couple of seconds).

Reply:
Ah, wait, you're saying we could keep the `After`, but remove `Wants`?

Reply:
>Ah, wait, you're saying we could keep the After, but remove Wants? 

Yes.